### DISCUSSION TITLE
Option "all" in elements per page gone

### DISCUSSION THREAD

**Discussion by payback007** [jcorporation/myMPD#680] [Q&A] [answered]
myMPD version: 9.1.1

In previous version there was an option in "settings - apperance - list" to list all elements on one page. For now it is limited to 500. Is it possible to bring the option "all elements on one page" back beside to the actual settings?

Thank you very much for the great work on mympd anyway!

**Comment by jcorporation** [comment, marked as answer]
Thank you!

The option "all" has lead to performance issues and browser freezes. Thats why it was removed.
